containers/ddev-php-base/Dockerfile

```diff
@@ -79,7 +79,7 @@ RUN apt-get -qq install --no-install-recommends --no-install-suggests -y \
     postgresql-client \
     sqlite3
 
-RUN npm config set unsafe-perm true && npm install --global gulp-cli yarn
+RUN npm install --unsafe-perm=true --global gulp-cli yarn
```

Comment:
The reality is I'm not completely sure why we even install nodejs this early, when we're going to do it later in post-build. 

At this point, though, we're building with NODE_LTS=16, so is node 16 affected by this change?

Reply:
I don't see this error on node 16 - but I also haven't tested this PR to know it fixes things for us.

Reply:
Maybe we should remove this step here then?